images/builder/docker/custom-docker-builder/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM openshift/origin-base
+FROM openshift/origin-base:v1.5
```

Comment:
We do not have the ability to alter FROM as of now. But I see the point of putting specific FROM for versioned images. @mohammedzee1000 these dockerfiles are specific to v1.5 right? then how this is getting build now? taking base as latest?